src/nidcpower/metadata/functions_addon.py

```diff
@@ -418,6 +417,52 @@
             'note': 'This function is not supported on all devices.',
         },
     },
+    'FancyCreateAdvancedSequence': {
+        'codegen_method': 'python-only',
+        'returns': 'ViStatus',
+        'python_name': 'create_advanced_sequence',
+        'method_templates': [
+            { 'session_filename': 'fancy_advanced_sequence', 'documentation_filename': 'default_method', 'method_python_name_suffix': '', },
+        ],
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'Identifies a particular instrument session. **vi** is obtained from the niDCPower_InitExtCal or niDCPower_InitializeWithChannels function.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'SequenceName',
+                'type': 'ViString',
+                'documentation': {
+                    'description': 'Name of sequence.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'Sequence',
+                'type': 'ViInt32[]',  # Doesn't matter here
+                'documentation': {
+                    'description': 'Sequence.',
```

Comment:
We should better explain things here. If you want to leave it to its own PR (not a bad idea) please TODO it.

Reply:
Because we don't generate this method, this type doesn't matter.

Something will need to change for documentation though. I am not working on that yet, until we are sure of the implementation and how the API will look.